docs/data/joy/components/typography/TypographyTitleBody.js

```diff
@@ -9,152 +10,146 @@ export default function TypographyTitleBody() {
-        <SvgIcon size="lg">
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            fill="none"
-            viewBox="0 0 24 24"
-            strokeWidth={1.5}
-            stroke="currentColor"
-          >
-            <path
-              strokeLinecap="round"
-              strokeLinejoin="round"
-              d="M3.75 13.5l10.5-11.25L12 10.5h8.25L9.75 21.75 12 13.5H3.75z"
-            />
-          </svg>
-        </SvgIcon>
-        <div>
-          <Typography level="title-lg">
-            <i>title-lg</i>: Title of the component
-          </Typography>
-          <Typography level="body-md">
-            <i>body-md</i>: This is the description of the component that contain
-            some information of it.
-          </Typography>
-          <Typography level="body-sm">
-            <i>body-sm</i>: Metadata, e.g. a date.
-          </Typography>
-        </div>
-      </Stack>
-
-      <Stack direction="row" spacing={1.5}>
-        <SvgIcon>
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            fill="none"
-            viewBox="0 0 24 24"
-            strokeWidth={1.5}
-            stroke="currentColor"
-          >
-            <path
-              strokeLinecap="round"
-              strokeLinejoin="round"
-              d="M3.75 13.5l10.5-11.25L12 10.5h8.25L9.75 21.75 12 13.5H3.75z"
-            />
-          </svg>
-        </SvgIcon>
-        <div>
-          <Typography level="title-md">
-            <i>title-md</i>: Title of the component
-          </Typography>
-          <Typography level="body-md">
-            <i>body-md</i>: This is the description of the component that contain
-            some information of it.
-          </Typography>
-          <Typography level="body-sm">
-            <i>body-sm</i>: Metadata, e.g. a date.
-          </Typography>
-        </div>
-      </Stack>
-
-      <Stack direction="row" spacing={1.5}>
-        <SvgIcon size="sm">
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            fill="none"
-            viewBox="0 0 24 24"
-            strokeWidth={1.5}
-            stroke="currentColor"
-          >
-            <path
-              strokeLinecap="round"
-              strokeLinejoin="round"
-              d="M3.75 13.5l10.5-11.25L12 10.5h8.25L9.75 21.75 12 13.5H3.75z"
-            />
-          </svg>
-        </SvgIcon>
-        <div>
-          <Typography level="title-sm">
-            <i>title-sm</i>: Title of the component
-          </Typography>
-          <Typography level="body-sm">
-            <i>body-sm</i>: This is the description of the component that contain
-            some information of it.
-          </Typography>
-          <Typography level="body-xs">
-            <i>body-xs</i>: Metadata, e.g. a date.
-          </Typography>
-        </div>
-      </Stack>
+      <Card>
+        <Stack direction="row" spacing={1.5}>
+          <SvgIcon size="lg">
+            <svg
```

Comment:
Maybe we can extract the `svg` from here and put it above as a component to avoid repetition?
Something like this, above the `TypographyTitleBody` component definition:

```
function BoltIcon(){
  return (
     <svg>...</svg>
);
```

And then we can do this:
```
<SvgIcon size="lg">
   <BoltIcon/>
</SvgIcon>
```

Reply:
However, in hindsight, now that we've deleted the "Each level are designed to fit perfectly with the same size of the `SvgIcon` component." part, I wonder if having the icon here is beneficial at all — maybe we can remove it for the sake of simplicity? Not seeing a lot of value in maintaining it!